MoviesGo/MoviesGo/DeteilModel/View/DescriptionViewController.swift

```diff
@@ -11,72 +11,68 @@ final class DescriptionViewController: UIViewController {
         case releaseDate
     }
 
-    // MARK: - PRIVATE PROPERTY
+    // MARK: - Private properties
 
-    private let descriptionTabelView = UITableView()
+    private let tabelView = UITableView()
     private let typeCells: [TypeCell] = [.poster, .title, .releaseDate, .overview]
-    private var moviesImage = UIImageView()
-    private var nameMoviesLabel = UILabel()
-    private var descriptionMoviesText = UITextView()
-    private var releaseDate = UILabel()
-    private var networkLayer = NetWorkLayer()
-    private var detailsMovie: DetailsMovie? {
-        didSet {
-            descriptionTabelView.reloadData()
-        }
-    }
-
-    var id: Int?
-    var onSelectedID: ((Int) -> Void)?
+    private var viewModel: DetailsViewModelProtocol!
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        navigationController?.overrideUserInterfaceStyle = .dark
-        tabBarController?.tabBar.barTintColor = .white
-        navigationController?.navigationBar.prefersLargeTitles = true
-        navigationItem.title = detailsMovie?.title
-        createDescriptionTabelView()
-        loadData()
+        createTitleNC()
+        createTabelView()
+        reloadDataView()
+    }
+
+    func injectionViewModel(viewModel: DetailsViewModelProtocol) {
+        self.viewModel = viewModel
     }
 
-    private func loadData() {
-        if let id = id {
-            networkLayer.fetchData(DetailsMovie.self, methodStr: "movie/\(id)") { [weak self] result in
-                switch result {
-                case let .failure(error):
-                    switch error {
-                    case let .failure(error):
-                        print("Ошибка связи с Бэк Error: \(error.localizedDescription)")
-                    case .failureDecode:
-                        print("Ошибка декодирования")
-                    case .notData:
-                        print("Отсуствуют данные")
-                    }
-                case let .success(data):
-                    self?.detailsMovie = data
-                }
+    // MARK: - Private functions
+
+    private func reloadDataView() {
+        viewModel.loadData()
+        viewModel.updateData = { [weak self] in
+            guard let self = self else { return }
+            DispatchQueue.main.async {
+                self.tabelView.reloadData()
+            }
+        }
+
+        viewModel.showError = { [weak self] errorText, isReload, completion in
+            let alert = UIAlertController(title: errorText, message: nil, preferredStyle: .alert)
+            alert.addAction(UIAlertAction(title: "Ok", style: .destructive))
+            if isReload {
+                alert.addAction(UIAlertAction(title: "Reload", style: .default) { _ in
```

Comment:
```suggestion
                alert.addAction(UIAlertAction(title: "Reload", style: .default) { _ in
```
"Reload" в константу